packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Added
+- SF-1125: Added option for parsing `collection` and `area` through URL on details pages via configurable key-names.
+  - This new option can be leveraged via query string as follows: `?area=Production&collection=productsLeaf`
+    - Where `Production` and `productsLeaf` are examples of collection and area, respectively.
```

Comment:
Add a note re: the default values (shown above). For example, something to the effect of:
- Where 'area' and 'collection' are the default key-names for `area` and `collection`, respectively.

Reply:
Added.